tests/ipv6-evasion/ipv6-covert-dstopts/README.md

```diff
@@ -0,0 +1,7 @@
+# Description
+
+Test detection of covert channel within IPv6 Destination Options Header
+
```

Comment:
This attack looks legit to me and Suricata works with this test.

Reply:
it does detect it, yes. but not sure if we could improve the rule.